packages/office-ui-fabric-react/src/components/Dropdown/Dropdown.tsx

```diff
@@ -66,7 +66,9 @@ export class Dropdown extends BaseComponent<IDropdownProps, any> {
 
     return (
       <div ref='root'>
-        <label id={ id + '-label' } className='ms-Label' ref={ (dropdownLabel) => this._dropdownLabel = dropdownLabel } >{ label }</label>
+        { label && (
+          <label id={ id + '-label' } className='ms-Label' ref={ (dropdownLabel) => this._dropdownLabel = dropdownLabel } >{ label }</label>
```

Comment:
ref should be this._resolveRef('_dropdownLabel'), but that can change later, other than that looks good.